generate-stackbrew-library.sh

```diff
@@ -2,8 +2,7 @@
 set -eu
 
 declare -A aliases=(
-	[3.6-rc]='rc'
```

Comment:
When you come back around to updating this, can you change this line to be `[3.7-rc]='rc'` instead so we're ready for the next version's RC cycles instead of removing this line entirely? :pray: (it's harmless to have an unused line here)